### ISSUE TITLE
Feature Request: Specify quality profile when selecting movie/show

### ISSUE THREAD

**Issue by nrgbistro** [darkalfx/requestrr#11]
It would be very nice to see this so I can request different shows or movies in different qualities depending on which show it is. I know there are options in the bot configuration but it would be awesome to have an on-the-fly option as well.

**Comment by eckozen84** [comment]
+1 Definitely would love to have this feature

**Comment by darkalfx** [comment]
It’s pretty much the highest priority feature at the moment, should be out in a month or so.

**Comment by NoName95x** [comment]
+1 , watching this issue with interest.

**Comment by ricain59** [comment]
Any way this request implemented? Thanks you for your work.

**Comment by darkalfx** [comment]
Yes, I am aiming to have it out for Radarr/Sonarr this weekend, Overseerr and Ombi should come the following week.

**Comment by kylerm42** [comment]
Are there any updates on this? If there's been any progress on a branch anywhere, I wouldn't mind taking a look and helping out. This feature would be a game-changer for me so everybody isn't downloading 4K all the time.

**Comment by darkalfx** [comment]
> Are there any updates on this? If there's been any progress on a branch anywhere, I wouldn't mind taking a look and helping out. This feature would be a game-changer for me so everybody isn't downloading 4K all the time.

This has been done for Sonarr/Radarr/Overseerr, only Ombi is left but that will come with the multi-instance update.

**Comment by thrashertn** [comment]
Is this feature implemented yet? How to enable it if so?

**Comment by RobinHoutevelts** [comment]
Also unclear to me how to enable this.  
I have overseerrr with multiple radarr servers, each with a different profile.

When requesting in discord I don't see the option to select a quality profile.

**Comment by darkalfx** [comment]
Use categories for this
